### PR TITLE
net-mail/notmuch: add 0.36

### PR DESCRIPTION
**Pull request by Austin-Ray**
Closes: https://bugs.gentoo.org/841215
Signed-off-by: Austin Ray <austin@austinray.io>

### CODE REVIEW

File: net-mail/notmuch/notmuch-0.36.ebuild [hunk 1]
```diff
@@ -0,0 +1,302 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (gyakovlev) [inline net-mail/notmuch/notmuch-0.36.ebuild:4]: 8?

File: net-mail/notmuch/notmuch-0.36.ebuild [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+DISTUTILS_OPTIONAL=1
+NEED_EMACS="24.1"
+PYTHON_COMPAT=( python3_{7..10} pypy3 )
```
Comment (gyakovlev) [inline net-mail/notmuch/notmuch-0.36.ebuild:8]: remove 3.7, add 3.11 (if tests pass ofc)

File: net-mail/notmuch/notmuch-0.36.ebuild [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+DESCRIPTION="Thread-based e-mail indexer, supporting quick search and tagging"
+HOMEPAGE="https://notmuchmail.org/"
+SRC_URI="https://notmuchmail.org/releases/${P}.tar.xz
+	test? ( https://notmuchmail.org/releases/test-databases/database-v1.tar.xz )"
+
+LICENSE="GPL-3"
+# Sub-slot corresponds to major wersion of libnotmuch.so.X.Y. Bump of Y is
+# meant to be binary backward compatible.
+SLOT="0/5"
```
Comment (gyakovlev) [inline net-mail/notmuch/notmuch-0.36.ebuild:20]: just in case, does the soversion still matches?

**Review by gyakovlev — CHANGES_REQUESTED** [review]
I'll give it a test, but seems it's ok to bump eapi and python support.

File: net-mail/notmuch/notmuch-0.36.ebuild (hunk 1 above)
Comment (Austin-Ray) [reply net-mail/notmuch/notmuch-0.36.ebuild]: Bumped version

File: net-mail/notmuch/notmuch-0.36.ebuild (hunk 2 above)
Comment (Austin-Ray) [reply net-mail/notmuch/notmuch-0.36.ebuild]: Removed 3.7. But I'm not sure how to emerge 3.11 on my system. I'll try again tomorrow, but if you have a link for install docs, I'd appreciate it.

File: net-mail/notmuch/notmuch-0.36.ebuild (hunk 3 above)
Comment (Austin-Ray) [reply net-mail/notmuch/notmuch-0.36.ebuild:20]: Yes, it creates `libnotmuch.so.5.6.0` so the major version is still the same.

**Pull request closed without merging**